modules/articles/server/controllers/articles.server.controller.js

```diff
@@ -35,6 +35,7 @@ exports.read = function (req, res) {
 
   // Add a custom field to the Article, for determining if the current User is the "owner".
   // NOTE: This field is NOT persisted to the database, since it doesn't exist in the Article model.
+  /* eslint no-unneeded-ternary: 0 */
```

Comment:
Because then if `req.user` is `undefined` we get returned `undefined` instead of `false`. That was failing one test.